NiftyPET/NIPET
# add tests & fixes

- [x] add amyloid pipeline tests
  + [x] `histogramming`
  + [ ] `basic_recon` <- was passing, now failing after updated scatter method; needs new [Zenodo reference data][zenodo]
  + [ ] `align_mumap[spm]` <- failing due to wrong mumap dims
  + [ ] `align_mumap[niftyreg]` <- failing due to wrong mumap dims
- [x] improve `pytest` framework
- [x] add cloud testing
- [x] add coverage reporting
- [x] fix CMake warnings
- [x] fix multi-array `npy` => `npz`
- [x] replace deprecated `nii.get_data` => `nii.get_fdata`
- [x] misc build & packaging improvements
- [x] misc formatting improvements

How to run tests:

- [`Ab_PET_mMR_test`][zenodo] should exist in either `$HOME` or `$DATA_ROOT`

```bash
git checkout test
pip install -e .[dev]
pytest
pytest -k "align"  # only tests with "align" in the name
```

[zenodo]: https://zenodo.org/record/3877529
